multisig/README.md

```diff
@@ -75,17 +75,57 @@ rustup target add wasm32-unknown-unknown
 
 ## Usage
 
+Before deploying the contract, you need to collect all public keys that it will be initialized with.
+
 Commands to deploy and initialize a 2 out of 3 multisig contract via `near repl`:
 
 ```javascript
-const tx = nearlib.transaction.createTransaction(
-[
-    nearlib.transactions.createAccount(),
-    nearlib.transactions.transfer(10),  
-    nearlib.transactions.addKey(`<1st_public_key>`),
-    nearlib.transactions.addKey(`<2nd_public_key>`),
-    nearlib.transactions.addKey(`<3nd_public_key>`),
-    nearlib.transactions.deploy(),
-    nearlib.transactions.functionCall("new", {"num_confirmations": 2}, 0, 10000000000000000),
-]);
+const fs = require('fs');
+const account = await near.account("illia");
+const contractName = "multisig.illia";
+const methodNames = ["add_request","delete_request","confirm"];
+const newArgs = {"num_confirmations": 2};
+const result = account.signAndSendTransaction(
+    contractName,
+    [
+        nearAPI.transactions.createAccount(),
+        nearAPI.transactions.transfer("100000000000000000000000000"),  
+        nearAPI.transactions.addKey(
+            nearAPI.utils.PublicKey.from("Eg2jtsiMrprn7zgKKUk79qM1hWhANsFyE6JSX4txLEuy"),
+            nearAPI.transactions.functionCallAccessKey(contractName, methodNames, "0")),
```

Comment:
I think it should be `null` instead of `"0"`
```suggestion
            nearAPI.transactions.functionCallAccessKey(contractName, methodNames, null)),
```